PacketDotNet/VxlanFields.cs

```diff
@@ -0,0 +1,44 @@
+/*
+This file is part of PacketDotNet.
+
+This Source Code Form is subject to the terms of the Mozilla Public
+License, v. 2.0. If a copy of the MPL was not distributed with this
+file, You can obtain one at https://mozilla.org/MPL/2.0/.
+*/
+
+namespace PacketDotNet
+{
+    /// <summary>
+    /// Defines the lengths and positions of the vxlan fields within
+    /// a vxlan packet
+    /// </summary>
+    public struct VxlanFields
+    {
+        /// <summary>Length of the Flags in bytes.</summary>
+        public static readonly int FlagsLength = 1;
+
+        /// <summary>Position of the VNI.</summary>
+        public static readonly int FlagsPosition = 0;
+
+        /// <summary>Length of the Reserved in bytes.</summary>
+        public static readonly int Reserved1Length = 3;
+
+        /// <summary>Length of the VNI in bytes.</summary>
+        public static readonly int VniLength = 3;
+
+        /// <summary>Position of the VNI.</summary>
+        public static readonly int VniPosition = 4;
+
+        /// <summary>Length of the Reserved in bytes.</summary>
+        public static readonly int Reserved2Length = 1;
+
+        /// <summary>Length of the vxlan header in bytes.</summary>
+        public static readonly int HeaderLength = FlagsLength + Reserved1Length + VniLength + Reserved2Length;
+
+        /// <summary>IANA assigned Vxlan destination UDP port by default.</summary>
+        public static readonly ushort DefaultUdpDstPortForVxlan = 4789;
+
+        /// <summary>The Vxlan port should be configurable according to RFC 7348.</summary>
+        public static ushort UdpDstPortForVxlan = DefaultUdpDstPortForVxlan;
```

Comment:
This seems useless.

Reply:
Actually custom port is very common in industrial scenarios. Especially for early systems that are still running. I suggest to keep it, as it says in standard: https://www.rfc-editor.org/rfc/rfc7348.html#page-10:~:text=Destination%20Port%3A%20IANA,SHOULD%20be%20configurable.

Reply:
OK, I thought it referred to the default one initially, it's fine then :)